src/lib/PDF.cpp

```diff
@@ -175,7 +221,50 @@ pair<double, double> PDF::uncMin(double dchi) {
   }
 
   // Loop over each point of the PDF to find the maximum error
-  for (size_t k = ib; k < xaxis.size(); k++) {
+  for (size_t k = ib; k < xaxis.size() - 1; k++) {
```

Comment:
Is this the key line for fixing the original issue which was going to an index one too high?

Reply:
Not the only one no. Others have now disappeared because of code refactoring.